curriculum/challenges/english/03-front-end-libraries/jquery/remove-an-element-using-jquery.md

```diff
@@ -12,7 +12,7 @@ Now let's remove an HTML element from your page using jQuery.
 
 jQuery has a function called `.remove()` that will remove an HTML element entirely
 
-Remove element `target4` from the page by using the `.remove()` function.
+Remove element with id `target4` from the page by using the `.remove()` function.
```

Comment:
```suggestion
Remove the `#target4` element from the page by using the `.remove()` function.
```

Reply:
@chinmaykunkikar if you agree with Shaun's suggestion above, could you go ahead and commit it?